StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewModel.swift

```diff
@@ -12,6 +12,43 @@ import Foundation
 @_spi(STP) import StripeUICore
 import UIKit
 
+// calculate the resulting color of the mask overlaying the component background
+private func disabledBackgroundColor(componentBackground: UIColor) -> UIColor {
```

Comment:
This feels quite complex for generating a background color. What's the formula for this in english? Is it really necessary?

Reply:
If we keep it we should put it in it's own file or find an appropriate file and make it an extension on `UIColor`.